### PR TITLE
feat: openapi for shares endpoint

### PR DESCRIPTION
**Pull request by VK-RED**
This PR adds openapi support for the following endpoints 

image

### CODE REVIEW

**Comment by VK-RED** [comment]
## Screenshots

### Create
create1
create2

### GetOne
getOne

### GetMany
getMany1
getMany2

### Update
update1
update2

### Delete
delete

**Comment by VK-RED** [comment]
@dahal you can check it out !!

**Comment by dahal** [comment]
This PR is blocked by #407 , I have updated few things on prisma pagination proxy, lets wait for it to get merged before merging this one.

**Review by dahal — APPROVED** [review]

**Pull request merged**
